portfolio/src/main/webapp/index.html

```diff
@@ -73,6 +73,11 @@ <h2>Fact Hat</h2>
             </div>
         </div>
         <button id="page_button" onclick="window.location.href='/images.html'"><strong>Go to Gallery!</strong></button>
+        <div id="section">
+            <p>Click here to see a random message:</p>
+            <button onclick="getRandomMessage()">Fetch a message!</button>
+            <div id="message-container"></div>
```

Comment:
2 space indentation

Reply:
I changed the indentation to 2 spaces. Similarly to the changes to the classes and ids, I only made changes to the changed lines in this pull request instead of the entire file because I want to minimize merge conflicts when merging the pull requests together.